crates/kwavers-python/tests/test_typed_consumer.py

```
@@ -0,0 +1,146 @@
+    # configured [tool.mypy] settings (python_version 3.8, strict).
+    cmd = [
+        sys.executable,
+        "-m",
+        "mypy",
+        "--strict",
+        "--python-version",
+        "3.8",
+        str(PYTHON_SOURCE / "pykwavers"),
+        str(FIXTURE),
+    ]
+    proc = subprocess.run(cmd, capture_output=True, text=True, cwd=ROOT)
```

Ruff (0.16.1)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)